libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Recognizers/RecognizerSet.cs

```diff
@@ -0,0 +1,186 @@
+﻿// Licensed under the MIT License.
+// Copyright (c) Microsoft Corporation. All rights reserved.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text.RegularExpressions;
+using System.Threading;
+using System.Threading.Tasks;
+using Antlr4.Runtime;
+using Microsoft.Bot.Schema;
+using Microsoft.Recognizers.Text.NumberWithUnit;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Bot.Builder.Dialogs.Adaptive.Recognizers
+{
+    /// <summary>
+    /// RecognizerSet - InputRecognizer which is the union of of multiple recognizers into one RecognizerResult.
+    /// </summary>
```

Comment:
Might want to document how you merge in particular if intents collide you pick the max score and if entities collide you union into the entities array.